frontend/src/FoodType.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+const FoodTypeItem = ({ id, food }) => (
+  <a href={`/${id}`}>
+    <img src={`/${id}-icon.svg`} />
+    <p>{food}</p>
+  </a>
+);
+
+const FoodTypes = () => {
+  const FoodTypeList = [
+    {
+      id: 'western',
+      food: '양식',
+    },
+    {
+      id: 'franchise',
+      food: '프랜차이즈',
+    },
+    {
+      id: 'korean',
+      food: '한식',
+    },
+    {
+      id: 'japanese',
+      food: '일식',
+    },
+    {
+      id: 'vegan',
+      food: '채식',
+    },
+    {
+      id: 'dessert',
+      food: '디저트',
+    },
+  ];
+
+  return (
+    <div id="foodtype">
+      {FoodTypeList.map(FoodType => (
+        <FoodTypeItem id={FoodType.id} food={FoodType.food} />
+      ))}
```

Comment:
```suggestion
      {FoodTypeList.map({id, food} => (
        <FoodTypeItem id={id} food={food} />
      ))}
```
destructuring으로 중복 없이 편하게 사용 가능합니다~
- cf. [poiemaweb - 객체 디스트럭쳐링](https://poiemaweb.com/es6-destructuring#2-%EA%B0%9D%EC%B2%B4-%EB%94%94%EC%8A%A4%ED%8A%B8%EB%9F%AD%EC%B2%98%EB%A7%81-object-destructuring)